dashboard/test/controllers/regional_partners_controller_test.rb

```diff
@@ -85,4 +85,56 @@ def self.test_workshop_admin_only(method, action, params = nil)
     assert @regional_partner.program_managers.exists?(teacher.id)
     assert teacher.program_manager?
   end
+
+  test 'add mapping updates regional partner mapping' do
+    sign_in @workshop_admin
+    post :add_mapping, params: {id: @regional_partner.id, region: '12345'}
+    assert @regional_partner.mappings.length == 1
```

Comment:
Will this test conflict with other tests? If it runs first, it will change `@regional_partner.mappings` value, which will affect assertions in other tests.

Reply:
Good point--I had assumed setup_all was run before each test but it looks like that is actually not the case

Reply:
fixed by creating @regional_partner in a setup method so it's clean every time